lib/dal/src/cached_module.rs

```diff
@@ -246,14 +245,10 @@ impl CachedModule {
 
                 let client = module_index_client.clone();
                 join_set.spawn(async move {
-                    let module_id = module.id.to_owned();
+                    let module_id = Ulid::from_string(&module.id).unwrap_or_default().into();
```

Comment:
I left Ulid::from_string() here because we use unwrap_or_default(), which rightfully does not exist on ModuleId. We arguably should not be doing this, but I really don't want to change behavior in this PR!

Reply:
Good call